app.py

```diff
@@ -18,31 +18,43 @@
 
 def get_layout():
     boroughs = mapping.borough_area()
+    inventory_df = read_data.get_rental_inventory()
     inventory_graph = [
         dcc.Dropdown(sorted(boroughs.keys()),
                      placeholder='Select a borough', id='borough-dropdown'),
         dcc.Dropdown(id='areas-dropdown', placeholder='Areas'),
-        dcc.Graph(id='area-inventory')
+        dcc.Graph(id='area-inventory-line')
+    ]
+    inventory_sunburst = [
+        dcc.Dropdown(sorted(list(set(inventory_df['date'].dt.year))), id='year-dropdown',
+                     placeholder='Select a year'),
+        dcc.Graph(id='area-inventory-sunburst')
     ]
     return html.Div([
         navbar,
         html.H2('Hello QQ'),
         dbc.Tabs(
             [
-                dbc.Tab(inventory_graph, label='Rental Listings'),
-                dbc.Tab("Placeholder", label='Tab2')
+                dbc.Tab(inventory_graph, label='Rental Listings by Borough'),
+                dbc.Tab(inventory_sunburst, label='Rental Listings by Year')
             ]
         ),
     ])
 
 
 app.layout = get_layout()
 
+# callbacks ---------------------------------------------
```

Comment:
Did you add these comments for documentation?  You could probably just give your callbacks descriptive names so we can tell what they are doing.

Reply:
yes, I added it. it just means that below is a callback section.